### ISSUE TITLE
Add both annotation and gene product count information to pager widget in search widget

### ISSUE THREAD

**Issue by kltm** [geneontology/amigo#276]
A stronger version involving #270 and #275.

The idea is that in the search pager where the total are, you could see both the total annotation and the total gene products at the same time.

This would require the full switch to #104 first, so we could just have an AmiGO-specialized pager widget that would not interfere with the general widget usability.

The more I think about this, the less I'm sure it could work (may drp to "wontfix"). What would filters even mean in a case like this where you are browsing two different aspects at the same time?

**Comment by ValWood** [comment]
Hi Seth, 
the way I envusage, in the default view you would still be browsing annotations, you would therefore be able to filer for example on evidence , which you cannot do in the gene product view.
However, the number reported would include the gene products as well as the annotations.

Maybe this should be presented as:
Total associations: 83996  (to 51848 gene products)
The primary data type would still be annotations, the number of gene products is just a piece of additional data to make it clearer what the user is looking at.
It should then be more obvious to people that if they want the actual gene products via this route they need to download the  file and filter on unique gene label (that is also a bit challenging for human...).

**Comment by kltm** [comment]
Yes, however, the types of filters that can be applied to gene products and annotations may not overlap, which would lead to some bad situations. For example, if I'm browsing annotations and filter on evidence, the total count for annotations drops, but what happens to the gene product count? Well, since there is (currently) no stored evidence information grouped at the gene product level, we can't know--it may or may not have made a difference, the number may or may not be correct.

To be able to give two numbers at the same time, you essentially have to continuously calculate a gene product "view" from the annotation data on-the-fly. We'll probably be able to do something like that in the future with features present in more recent versions of solr, but right now it's not something that's easily obtainable.
